config/formats.js

```diff
@@ -407,51 +407,71 @@ let Formats = [
 		column: 2,
 	},
 	{
-		name: "[Gen 7] Metagamiate",
-		desc: `Every Pok&eacute;mon gains an intrinsic -ate ability matching its primary type, or its secondary type if shiny.`,
+		name: "[Gen 7] Tier Shift",
+		desc: `Pok&eacute;mon get +10 to each stat per tier below OU they are in. UU gets +10, RU +20, NU +30, and PU +40.`,
 		threads: [
-			`&bullet; <a href="https://www.smogon.com/forums/threads/3604808/">Metagamiate</a>`,
+			`&bullet; <a href="https://www.smogon.com/forums/threads/3610073/">Tier Shift</a>`,
 		],
 
-		mod: 'metagamiate',
+		mod: 'gen7',
 		ruleset: ['[Gen 7] OU'],
-		banlist: ['Dragonite', 'Kyurem-Black'],
-		onModifyMovePriority: -1,
-		onModifyMove: function (move, source) {
-			if (move.type === 'Normal' && !['judgment', 'multiattack', 'naturalgift', 'revelationdance', 'technoblast', 'weatherball'].includes(move.id) && !move.isZ && move.category !== 'Status') {
-				if (source.hasAbility(['aerilate', 'pixilate', 'refrigerate', 'galvanize', 'normalize'])) return;
-				// @ts-ignore
-				move.isMetagamiate = true;
-				move.type = source.getTypes().length < 2 || !source.set.shiny ? source.getTypes()[0] : source.getTypes()[1];
+		banlist: ['Damp Rock', 'Deep Sea Tooth', 'Eviolite'],
+		onModifyTemplate: function (template, pokemon) {
+			let tsTemplate = Object.assign({}, template);
```

Comment:
? This line appears to do nothing at all?

Reply:
It's making a shallow clone, so that we can shallow clone `tsTemplate.baseStats` later.